solidity/test/token_stake/TestStakingGrant.js

```diff
@@ -656,5 +656,53 @@ describe('TokenStaking/StakingGrant', () => {
           "Not authorized"
         )
       })
+
+      it('lands back in the escrow when undelegated and recovered', async () => {
+        const redelegatedAmount = delegatedAmount
```

Comment:
Can we test it with a redelegation value different than the initial delegation value?

```suggestion
        const redelegatedAmount = delegatedAmount.subn(10)
```

Reply:
Sure, good idea!